app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), root_path, class: 'nav-link' %></li>
+    <h2>Sign up</h2>
+    <li><%= link_to 'Sign In', new_user_session_path, class: "nav-link" %></li>
+  </ul>
+</div>
+<div class="form-class">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <%= f.text_field :name, autocomplete: "name", autofocus: true, class: "", placeholder: "Enter Name" %>
+
+  <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Enter Email" %>
+    <% if @minimum_password_length %>
+    <% end %>
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Enter Password" %>
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm Password" %>
+    <%= f.submit "Sign up" %>
+<% end %>
+</div>
```

Comment:
- [ ] Kindly design the `Signup` and `Login` pages of your application to look identical to that of the Behance design used in this project. Designing a page based on a wireframe or sample layout ensures consistent and accurate implementation of the intended user experience, reducing ambiguity and misinterpretation. In the real world, following these visual guidelines enhances collaboration among team members, streamlines development, and helps deliver a product that aligns closely with stakeholders' expectations. 💻 

| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/misalazeem/Budget-App/assets/77106962/c6340530-1ea8-4395-bef5-5c12f1bda686) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/1167219c-4a3b-48e4-b19d-22ebb21c5edd) |

#### Hint: Please pay attention to the height and width of the Navigation bar and the design layout of the form especially the border-colour, placeholder colour and width of the form inputs. 👍 🆒